DataFormats/L1TMuon/interface/EMTFOutput.h

```diff
@@ -70,7 +71,8 @@ namespace l1t {
     emtf::EventTrailer GetEventTrailer()    { return EventTrailer;  };
     emtf::MTF7Trailer GetMTF7Trailer()      { return MTF7Trailer;   };
     emtf::AMC13Trailer GetAMC13Trailer()    { return AMC13Trailer;  };
-    const uint64_t Dataword() const     { return dataword; };
+    const int Format_Errors() const { return format_errors; };
+    const uint64_t Dataword() const { return dataword; };
```

Comment:
The point here is the code in 80X is diverging from 81X on account of a separate PR as Mike mentioned. (#13891)
If we were to compare the L1 packages at a point in the 81X history before this other PR, I suspect no discrepancy would be found.